float64 lat
float64 lon
float64 alt
float32 yaw_rad
uint32 mission_type